VectorRumble.Shared/Screens/GameplayScreen.cs

```diff
@@ -158,35 +158,45 @@ private void DrawHud(float elapsedTime)
 
             Vector2 position = new Vector2(128, 64);
             int offset = (1280) / 5;
+            float scale = 1.0f;
 
-            for (int i = 0; i < World.Ships.Length; ++i)
+            var playerIndexes = Enum.GetValues(typeof(PlayerIndex)).Cast<PlayerIndex>();

```

Comment:
you can just use a hard cast here

```
(PlayerIndex[])Enum.GetValues (typeof (PlayerIndex))
```

Then there is not need to call into `Linq`.

Reply:
Thanks Done!